core/src/main/scala/kafka/server/AutoTopicCreationManager.scala

```diff
@@ -50,6 +50,10 @@ trait AutoTopicCreationManager {
 
 object AutoTopicCreationManager {
 
+  /**
+   * @deprecated use [[apply(kafka.server.KafkaConfig, kafka.server.MetadataCache, scala.Option, scala.Option, scala.Option, scala.Option, org.apache.kafka.coordinator.group.GroupCoordinator, kafka.coordinator.transaction.TransactionCoordinator)]]
+   */
+  @deprecated("Use the alternative apply method", "3.8.0")
```

Comment:
I could alternatively delete this method altogether.

Reply:
If it's not used anymore or if it's easy to update the remaining callers, let's delete this method